src/styles/variables.scss

```
@@ -8,4 +8,5 @@
   --accent-color3: #c7f9cc;
   --accent-color4: #174444;
   --static-color: #f7f7f7;
+  --robotoFont: "Roboto";
```

```suggestion
  --font-family: "Roboto";
```
Please change to this